modules/contributor-guide/partials/con_che-extension-points.adoc

```diff
@@ -253,7 +253,7 @@ Note that kbd:[Ctrl+f] kbd:[Alt+b] defines a _chord_: the command is invoked aft
 
 Plug-ins can add new *debuggers* by implementing the link:https://microsoft.github.io/debug-adapter-protocol/[Debugger Adapter Protocol] (DAP) for a specific debugger. The debugger can then register in the `contributes.debuggers` debuggers of a plug-in `package.json` file.
 
-// TODO: For instruction on how to add a debugger, see link:link[Adding support for a new debugger].
+// TODO: For instruction on how to add a debugger, see link.
```

Comment:
since the referenced link appears to be immediately below this comment, is the comment still needed?